scripts/UpdateUserListScript.sql

```diff
@@ -0,0 +1,11 @@
+/* mypath in line 7 is the folder path
+myfile.json in line 7 is the name of the users.json file
+userusers.json must be in newline delimetered format NDJson */
+
+CREATE table userListSbi(sbi numeric(9)); 
+CREATE TABLE tempUserJson (data jsonb); 
+\copy tempUserJson from 'mypath\users.json';
```

Comment:
Only comment from me is if we can speak to the CCoE team and verify whether they would rather us do the work upfront to generate the INSERT statements from the `user.json` file instead of them having to download the file as above. I think CCoE would prefer a ready made script they can just run when they are directly connected to the DB.

Otherwise, the above looks fine to me!